VMAccess/vmaccess.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+
+COMMAND="./vmaccess.py"
+PYTHON=""
+
+USAGE="$(basename "$0") [-h] [-i|--install] [-u|--uninstall] [-d|--disable] [-e|--enable] [-p|--update] -- program to invoke the extension script
+
+where:
+    -h              show this help text
+    -i|--install    install the extension
+    -u|--uninstall  uninstall the extension
+    -d|--disable    disable the extension
+    -e|--enable     enable the extension
+    -p|--update     update the extension
+"
+
+function find_python(){
+    local python_version=$1
+
+    # Check if there is python defined.
+    which python > /dev/null 2>&1
+    return_code=`echo $?`
+
+    if [[ ${return_code} == 0 ]]
+    then
+        eval ${python_version}="python"
```

Comment:
The name of the variable 'python_version' is a little misleading... 'python_command' maybe?